tests/roles/mariadb_copy/tasks/env_vars_src.yaml

```diff
@@ -8,8 +8,7 @@
       SOURCE_MARIADB_IP={{ source_mariadb_ip|default(external_mariadb_ip) }}
       declare -A SOURCE_GALERA_MEMBERS
       SOURCE_GALERA_MEMBERS=(
-      ["{{ edpm_node_hostname }}"]={{ source_mariadb_ip|default(external_mariadb_ip) }}
-      # ...
+        {{  source_galera_members }}
```

Comment:
we are passing source_galera_members in vars.yaml https://review.rdoproject.org/r/c/rdo-jobs/+/53365/5/playbooks/data_plane_adoption/deploy_tripleo_run_repo_tests.yaml#179 

the resulting vars.yaml is there from last run https://logserver.rdoproject.org/32/52732/8/check/periodic-data-plane-adoption-multinode-osp-17-to-extracted-crc-no-ceph/0416d01/controller/data-plane-adoption-tests-repo/data_plane_adoption/vars.yaml 

```
edpm_computes: |
  ["np0004696832.localdomain"]="192.168.122.106"
  ["np0004696855.localdomain"]="192.168.122.107"
  ["np0004696871.localdomain"]="192.168.122.108"
source_galera_members: |
  ["np0004696804.localdomain"]="172.17.0.103"
  ["np0004696805.localdomain"]="172.17.0.104"
  ["np0004696806.localdomain"]="172.17.0.105"
edpm_nodes:
  np0004696832:
```

It also has the desired effect, the tests are targetting all three nodes now eg from the last run https://logserver.rdoproject.org/32/52732/8/check/periodic-data-plane-adoption-multinode-osp-17-to-extracted-crc-no-ceph/0416d01/controller/data-plane-adoption-tests-repo/data-plane-adoption/tests/logs/test_minimal_out_2024-06-05T07:14:35EDT.log

```
TASK [mariadb_copy : set src MariaDB copy shell vars] **************************
...
SOURCE_MARIADB_IP=172.17.0.103\ndeclare -A SOURCE_GALERA_MEMBERS\nSOURCE_GALERA_MEMBERS=(\n  [\"np0004696804.localdomain\"]=\"172.17.0.103\"\n[\"np0004696805.localdomain\"]=\"172.17.0.104\"\n[\"np0004696806.localdomain\"]=\"172.17.0.105\"\n\n)

...
TASK [mariadb_copy : check that the Galera database cluster members are online and synced] ***
...
"stdout_lines": ["Checking for the database node np0004696806.localdomain WSREP status Synced", "Checking for the database node np0004696804.localdomain WSREP status Synced", "Checking for the database node np0004696805.localdomain WSREP status Synced"]}
```